ext/fg/js/frontend.js

```diff
@@ -285,7 +290,8 @@ class Frontend {
             if (!hideResults && (!this.textSourceLast || !this.textSourceLast.equals(textSource))) {
                 searched = true;
                 this.pendingLookup = true;
-                hideResults = !await this.searchTerms(textSource) && !await this.searchKanji(textSource);
+                const focus = (type === 'mouse');
```

Comment:
I think it would make more sense to pass the type as-is and check if it's mouse later

Reply:
My reasoning was that it would be better to keep all reference to ```type```'s values (```'mouse'```, ```'touchMove'```, ```'touchStart'```) to the same file.

Reply:
Ah, I wasn't familiar with the code that much. Just in general I think that transforming a value for a single purpose in a state object can obscure its origin and it's also less useful if for some reason there must be special handling for touchMove and touchStart events in the future. Anyway I trust your reasoning here because you authored this feature (and it's not like I decide what's accepted anyway :laughing:)